src/Symfony/Component/Notifier/Bridge/ContactEveryone/Tests/ContactEveryoneTransportTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\ContactEveryone\Tests;
+
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\Notifier\Bridge\ContactEveryone\ContactEveryoneTransport;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Test\TransportTestCase;
+use Symfony\Component\Uid\Uuid;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+final class ContactEveryoneTransportTest extends TransportTestCase
+{
+    public function createTransport(HttpClientInterface $client = null): ContactEveryoneTransport
+    {
+        return new ContactEveryoneTransport('API_TOKEN', 'Symfony', 'Symfony', $client ?? $this->createMock(HttpClientInterface::class));
```

Comment:
```suggestion
        return new ContactEveryoneTransport('API_TOKEN', 'Symfony', 'Foo', $client ?? $this->createMock(HttpClientInterface::class));
```

Should be different to be more precise in tests

Reply:
Okay should I change the diffusionname (second argument) also to something like 'bar' ?